### DISCUSSION TITLE
Where to copy KML files for import?

### DISCUSSION THREAD

**Discussion by Shohreh** [organicmaps/organicmaps#878] [Q&A]
Hello,

The wiki is empty and searching "category:Q&A import kml" returned no hits.

What is Organic's alternative to Maps.me's \MapsWithMe directory in the phone's internal memory to copy and import KML files?

Thank you.

--
Edit: Even simpler. Unlike Maps.me, Organic lets you import files from anywhere.
![Organic import KML files](https://user-images.githubusercontent.com/1541861/126064889-1651a51d-4344-4c21-9b10-c5fb00a3ee78.png)

**Comment by rtsisyk** [comment]
You can import KML/KMZ/KMB (MapsMe binary format) from any Android folder. For example, try /sdcard/MapsWithMe bookmarks. 
OrganicMaps stores bookmarks files in a private directory. This is because of [new Google restrictions](https://developer.android.com/about/versions/11/privacy/storage). We investigate possible ways to export bookmarks in batch and/or store them in a public directory.
